medschool/src/parse.rs

```diff
@@ -279,48 +292,59 @@ fn dfs_fields<'a, const MAX_RECURSION_LEVEL: usize>(
 /// be where we resolve all references and return the same HashSet and let the caller
 /// decide what the root should be.
 #[tracing::instrument(level = "trace", ret)]
```

Comment:
```suggestion
#[tracing::instrument(level = Level::TRACE, ret)]
```

Also change it here, since you're touching this function.